pkg/providers/v1/aws_loadbalancer.go

```diff
@@ -68,6 +68,11 @@ var (
 	defaultHealthCheckPort         = "traffic-port"
 	defaultHealthCheckPath         = "/"
 
+	// We need the port as both string and int.
+	kubeProxyHealthCheckPort    = "10256"
+	kubeProxyHealthCheckPortInt = int32(10256)
```

Comment:
Just a thought, it should be pretty easy to do `strconv.Itoa(kubeProxyHealthCheckPort)` where needed.
```suggestion
	kubeProxyHealthCheckPort = int32(10256)
```